### PR TITLE
Support link share in amp-story

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: src/clipboard.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+export function copyTextToClipboard(doc, text) {
+  let copySuccessful = false;
+
+  const textarea = doc.createElement('textarea');
+
+  setStyles(textarea, {
+    'position': 'fixed',
+    'top': 0,
+    'left': 0,
+    'width': '50px',
```
Comment (newmuis) [inline src/clipboard.js]: Does the dummy textarea have to have a size?

File: src/clipboard.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  textarea.value = text;
+
+  doc.body.appendChild(textarea);
+
+  textarea.select();
+
+  try {
+    copySuccessful = doc.execCommand('copy');
+  } catch (e) {
+    // 🤷
```
Comment (newmuis) [inline src/clipboard.js]: lol

Might be more readable to set `copySuccessful = false` and just not initialize it above?

File: extensions/amp-story/0.1/share.js [hunk 3]
```diff
@@ -145,12 +175,49 @@ export class ShareWidget {
...
+  maybeAddLinkShareButton_() {
+    if (!isCopyingToClipboardSupported(this.win_.document)) {
+      return;
+    }
+
+    this.add_(
+        renderSimpleTemplate(this.win_.document, LINK_SHARE_ITEM_TEMPLATE));
+
+    // TODO(alanorozco): Listen for proper tap event (i.e. fastclick)
+    listen(
```
Comment (newmuis) [inline extensions/amp-story/0.1/share.js]: nit: Can you nest this less?

Comment (alanorozco) [reply extensions/amp-story/0.1/share.js]: Done

File: src/clipboard.js (hunk 1 above)
Comment (alanorozco) [reply src/clipboard.js]: Unfortunately :/

File: src/clipboard.js (hunk 2 above)
Comment (alanorozco) [reply src/clipboard.js]: 🤷

**Pull request merged**